intel-sgx/dcap-artifact-retrieval/src/provisioning_client/intel.rs

```diff
@@ -450,10 +443,17 @@ impl<'inp> ProvisioningServiceApi<'inp> for QeIdApi {
 
     fn build_request(&self, input: &Self::Input) -> Result<(String, Vec<(String, String)>), Error> {
         let api_version = input.api_version as u8;
-        let url = format!(
-            "{}/sgx/certification/v{}/qe/identity?update=early",
-            INTEL_BASE_URL, api_version,
-        );
+        let url = if let Some(tcb_evaluation_data_number) = input.tcb_evaluation_data_number {
+            format!(
+                "{}/sgx/certification/v{}/qe/identity?tcbEvaluationDataNumber{}",
+                INTEL_BASE_URL, api_version, tcb_evaluation_data_number
+            )
+        } else {
+            format!(
+                "{}/sgx/certification/v{}/qe/identity?update=early",
```

Comment:
Is there a use case where we would want to get the standard access instead of early without providing the TCB number?

Reply:
No as soon as the standard/early option was introduced, we always used the "early" one. It already used to be hard coded. Now we want either "early" or a specific TCB evaluation data number